backend/src/main/kotlin/org/jetbrains/kotlinconf/backend/Api.kt

```diff
@@ -1,15 +1,22 @@
 package org.jetbrains.kotlinconf.backend
 
-import io.ktor.http.*
-import io.ktor.server.application.*
-import io.ktor.server.auth.*
-import io.ktor.server.request.*
-import io.ktor.server.response.*
-import io.ktor.server.routing.*
-import io.ktor.util.date.*
-import org.jetbrains.kotlinconf.*
+import io.ktor.http.HttpStatusCode
+import io.ktor.server.application.ApplicationCall
+import io.ktor.server.auth.principal
+import io.ktor.server.request.receive
+import io.ktor.server.response.respond
+import io.ktor.server.routing.Route
+import io.ktor.server.routing.get
+import io.ktor.server.routing.post
+import io.ktor.server.routing.route
+import io.ktor.util.date.GMTDate
+import kotlinx.datetime.toInstant
+import org.jetbrains.kotlinconf.EVENT_TIME_ZONE
+import org.jetbrains.kotlinconf.FeedbackInfo
+import org.jetbrains.kotlinconf.VoteInfo
 import org.jetbrains.kotlinconf.Votes
-import java.time.*
+import java.time.Clock
+import java.time.LocalDateTime
```

Comment:
Is `java.time` still needed here?

Reply:
The backend code still uses `java.time` APIs in a bunch of places, I tried touching as little of the backend code as possible. Only edited the parts that were tightly coupled to the shared code.

We *could* rewrite the backend to be all `kotlinx.datetime` too, but it's not strictly necessary. Just wanted to have these types on the client side instead of using the Ktor `GMTDate` type everywhere.

@e5l to review the backend changes though, and whether we want more of them.

Reply:
Feel free to use kotlinx.datetime if it's possible